vehicle-python/vehicle/__init__.py

```diff
@@ -206,7 +227,7 @@ def _translate_at(self, contents: Dict[Any, Any]) -> str:
     def _translate_network(self, contents: Dict[Any, Any]) -> str:
         model = contents[0]
         input_losses = ", ".join([self._translate_expression(c) for c in contents[1]])
-        return f"{model}({input_losses}, training=True)"
+        return f"{model}(expand({input_losses}, axis=0), training=True)[0]"
```

Comment:
Could you add a comment here explaining why there is an `expand` inserted here?

Reply:
I genuinely cannot remember - I believe this was related to the shape of input to the NN which we had issues with but I don't want to add incorrect information here.

Reply:
Okay... In future when you add code like this that is likely to be mysterious in future, please can you add explanatory comments about it at the time! At the very least explaining what breaks if you remove it.